lib/rbs/inline/annotation_parser.rb

```diff
@@ -754,7 +754,14 @@ def parse_module_self(tokenizer)
         tree = AST::Tree.new(:module_self)
 
         tokenizer.consume_token!(K_MODULE_SELF, tree: tree)
-        tree << parse_type(tokenizer, tree)
+        while true
```

Comment:
```suggestion
        loop do
```

Reply:
I understand, but rbs-inline prefers this style:

```
tkomiya@tarf> grep -r 'while true' . | wc -l
       6
tkomiya@tarf> grep -r 'loop do' . | wc -l
       0
```

Reply:
Haha, yeah, I rarely write `loop do` over `while true`.
Not something with specific reason/background...